Policies/github-anti-bribery-statement.md

```diff
@@ -0,0 +1,50 @@
+---
+title: GitHub Anti-Bribery Statement
+---
+GitHub upholds our commitment to ethical business practices, including by prohibiting bribery and corruption. As [Transparency International](https://www.transparency.org/what-is-corruption#costs-of-corruption) explains, the costs of corruption are
```

Comment:
@leroy182 to ethical business practices. Including as transparency International. 
https://www.transparency.org/@leroy182